src/test/java/fr/nelson/you_are_the_hero/service/ScineServiceTest.java

```diff
@@ -0,0 +1,81 @@
+package fr.nelson.you_are_the_hero.service;
+import static org.junit.jupiter.api.Assertions.*;
+
+import fr.nelson.you_are_the_hero.repository.SceneRepository;
+import static org.mockito.Mockito.*;
+
+import fr.nelson.you_are_the_hero.model.dto.AddSceneDto;
+import fr.nelson.you_are_the_hero.model.Choice;
+import fr.nelson.you_are_the_hero.model.db.Scene;
+
+import java.util.Optional;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.util.ReflectionTestUtils;
+
+public class ScineServiceTest {
+    private SceneService sceneService;
+    private SceneRepository sceneRepository;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
you can replace this method by annoting SceneRepository with @Mock and SceneService with @InjectsMock

Reply:
My current task is focused on documentation, not unit testing. I believe the testing has already been completed.

Reply:
ok.